README.md

```diff
@@ -11,10 +11,11 @@ This implements ably broadcaster as a independent service provider library for [
 ## Features
 - Native ably-js support.
 - Low latency for client-events.
-- Disable public channels.
-- Update channel permissions for a user.
+- Update channel permissions for each user.
 - Update token expirty.
-- Fully compatible with [pusher/pusher-based](https://laravel.com/docs/9.x/broadcasting#client-ably) broadcasters, see [migrating section](#migrating-from-old-ablybroadcaster)
+- Disable public channels.
+- Fully compatible with pusher/pusher-compatible broadcasters, see [migrating section](#migrating-from-pusherpusher-compatible-broadcasters).
+- Integration tested using [ably sandbox](https://github.com/ably-forks/laravel-echo/tree/master/tests/ably).
```

Comment:
I don't think this is a feature, I think I would prefer to just remove this line

Reply:
Removed